org.hl7.fhir.publisher.core/src/main/resources/Messages_sv_SE.properties

```diff
@@ -0,0 +1,66 @@
+cs.summary.content=Innehåll
+cs.summary.defining_url=Definierande URL
+cs.summary.publisher=Utgivare
+cs.summary.source_resource=Källresurs
+sd.dict._x_note=[x] Anteckning
+sd.dict.alternate_names=Alternativa namn
+sd.dict.binding=Bindning
+sd.dict.choice_of=Val av
+sd.dict.comments=Kommentarer
+sd.dict.control=Kontroll
+sd.dict.default_value=Defaultvärde
+sd.dict.defined_on_this_element=Defined on this element
+sd.dict.example=Exempel
+sd.dict.fixed_value=Bestämt värde
+sd.dict.for_example_codes_see=För exempelkoder, se
+sd.dict.is_modifier=Är modifierare
+sd.dict.max_length=Maxlängd
+sd.dict.meaning_if_missing=Innebörd om saknas
+sd.dict.must_support=Måste stödja
+sd.dict.note=Anteckning
+sd.dict.pattern_value=Mönstervärde
+sd.dict.requirements=Krav
+sd.dict.snomed_ct_code=SNOMED-CT Code
+sd.dict.the_codes_shall_be_taken_from=Koder SKALL komma från
+sd.dict.the_codes_should_be_taken_from=Koder BORDE komma från
+sd.dict.type=Typ
+sd.dict.unordered=osorterad
+sd.head.card=Kard.
+sd.head.constraints_and_usage=Regler och användning
+sd.head.description_constraints=Beskrivning & regler
+sd.head.flags=Flaggor
+sd.head.name=Namn
+sd.head.type=Typ
+sd.inv.constraints=Regler
+sd.inv.details=Detaljer
+sd.inv.path=Sökväg
+sd.inv.requirements=Krav
+sd.summary.extensions=Extensions
+sd.summary.fixed_value_s_s=Bestämt värde: %s %s
+sd.summary.mandatory_s_s=Mandatory: %s %s
+sd.summary.must_support_s_s=Måste stödja: %s %s
+sd.summary.prohibited_s_s=Ej tillåten: %s %s
+sd.summary.slices=Skikt??
+sd.summary.structures=Strukturer
+sd.summary.summary=Sammanfattning
+sd.table.binding=Bindning
+sd.table.closed=Stängd
+sd.table.complex=Komplex
+sd.table.example=Exempel
+sd.table.fixed_value=Bestämt värde
+sd.table.max_binding=Max Binding
+sd.table.open=Öppen
+sd.table.slice=Skikt??
+sd.table.unknown_reference_to_s=Okänd referens till %s
+sd.table.unordered=Osorterad
+sd.tx.conformance=Följsamhet??
+sd.tx.path=Sökväg
+sd.tx.terminology_bindings_differential=Terminologibindningar (Differential)
+sd.tx.terminology_bindings=Terminologibindningar
+sd.tx.valueset_code=ValueSet / Code
+sd.tx.valueset=ValueSet
+vs.summary.defining_url=Definierande URL
+vs.summary.name=Namn
+vs.summary.publisher=Utgivare
+vs.summary.source_resource=Källresurs
+vs.summary.title=Titel
```

Comment:
If this file is just for testing, and the translations are not correct, we shouldn't be checking it into the main branch. It would probably be good idea to keep test code in branches. 
Please remove.